lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -42,8 +41,9 @@ const bool _useSkia =
     bool.fromEnvironment('FLUTTER_WEB_USE_SKIA', defaultValue: false);
 
 // If set to true, forces CPU-only rendering (i.e. no WebGL).
-const bool canvasKitForceCpuOnly =
```

Comment:
Add reason for using this e.g. testing etc

Reply:
Done